oaknational/Oak-Web-Application
# chore(deps): update Next and related packages

update Next and related packages

to test:
* a general run through of OWA just to check nothing weird happened
* Storybook should still build and work
